scripts/pcm-dpc/covid-19/dpc-covid19-ita_StatisticalVariable.mcf

```diff
@@ -0,0 +1,95 @@
+Node: dcid:count_MedicalConditionIncident_COVID_19_HospitalizedsWithSymptoms
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:HospitalizedsWithSymptoms
+
+Node: dcid:count_MedicalConditionIncident_COVID_19_IntensiveCare
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:IntensiveCare
+
+Node: dcid:count_MedicalConditionIncident_COVID_19_Hospitalized
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:Hospitalized
+
+Node: dcid:count_MedicalConditionIncident_COVID_19_PeopleInHomeIsolation
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:PeopleInHomeIsolation
+
+Node: dcid:count_MedicalConditionIncident_COVID_19_ActiveCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:count
+statType: dcs:measuredValue
+medicalStatus: dcs:ActiveCase
+
+Node: dcid:incrementalCount_MedicalConditionIncident_COVID_19_IncrementalActiveCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+incidentType: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+statType: dcs:measuredValue
+medicalStatus: dcs:IncrementalActiveCase
```

Comment:
I think just ActiveCase here. This is a very interesting schema dilemma, and I think you did the best way possible, especially given what already exists in the graph.

Reply:
I tried to use "medicalStatus: dcs:ActiveCase", but that causes this statvar to duplicate the cumulative ones. It seems that difference in "measuredProperty" does not help distinguish the statvars.

Reply:
done.